### ISSUE TITLE
Create better landing page

### ISSUE THREAD

**Issue by gregwhitworth** [WICG/open-ui#29]
It would be good for the landing page to do a good job of presenting the problem we're hoping to solve in a clean, simple and straightforward manner. Additionally help describe the relationship between the design systems and the web platform. We should make it easy to jump to the components. Additionally, it would be good to add the logos of companies that are involved in this project.

**Comment by gregwhitworth** [comment]
Initially assigning to Chris since I think he was the one that said he could tackle it.

**Comment by levithomason** [comment]
Any update @chrisdholt?  Really looking forward to this  😄 I'm also game for helping out if needed.

**Comment by gregwhitworth** [comment]
Gonna ping again on this @levithomason @chrisdholt, what's a solid ETA we can put this in? Let's say just an initial design comp? Does mid-May, say the 20th work?

**Comment by ststimac** [comment]
@gregwhitworth I'd be happy to pick this up if this isn't in progress yet.

**Comment by gregwhitworth** [comment]
@ststimac - Goodness I'd love it! We have discussed it numerous times being a need but unfortunately it kept getting bumped. So that would be wonderful if you can create some mockups. Let me know if you want to sync about primary content on the main page.

**Comment by ststimac** [comment]
@gregwhitworth yup we should sync! I've got some time tomorrow or Thursday morning then I'm taking a few days off. So if next week would be better that's fine too!

**Comment by gregwhitworth** [comment]
@ststimac hey what's the update on this?

Also, @levithomason and I were chatting and as you've seen regarding the Figma comps there are more than one demo that are beginning to reference Open UI. Right now the specs are heavily leaning towards developers (whether FE, component authors, or browser implementers) but it would be good for the landing page to call attention to the various benefits Open UI brings to each.

* Developer: "Some fancy words about how it's the definitive solution blah blah"
* Designer: "Leverage design tool kits to design alongside your developers with the same expectation of states, terminology, bla blah"

This could change the overall design of the specification itself to be more designer friendly rather than a bunch of events it could focus on the anatomy, use cases, and links to figma/sketch, etc. It would be good to be able to toggle this on the spec page as well.

**Comment by ststimac** [comment]
@gregwhitworth I sent you and @levithomason comps yesterday. Did they not come through?

**Comment by gregwhitworth** [comment]
@ststimac it got sucked in by a rule :) Thanks

**Comment by gregwhitworth** [comment]
@levithomason from the beginning have had a desire to be able usable by anyone (whether they work in web or not). Currently however all content is web focused we want to leave the door open for potential contributions to make specs tailored to folks that may be building out controls/components on other platforms. So let's make the subheading be:

> The missing industry standard definition of UI

As I noted in the Figma item, I'd like to see this comp initially show how it helps designers vs developers. Maybe this could be two button type toggle that is on each component and mirrored on the sub page specs.

The difference between the components under analysis and proposals is that analysis is solely research that has been collated for the component. As the component moves to proposal this moves up to the right hand side of the proposal to allow someone to still get at this research but alleviate confusion on their being to components with the same name just under different categories.

Overall it looks great, thanks taking this on.

**Comment by gregwhitworth** [comment]
@ststimac any update on the progress of this?

**Comment by ststimac** [comment]
@gregwhitworth no, I've not had a moment to make any updates to it unfortunately. Probably not until next week.

**Comment by brianespinosa** [comment]
I know the existing doc generation for this was basically just a copied and then cleaned up version of what Semantic UI React uses to generate docs. I am happy to help on the implementation side of things making whatever updates are needed. I think we have some MDX files being used now. If it turns out the designs could benefit from some custom components we can extend into our MDX files, I have been spending a lot of time recently doing this for my own org.

**Comment by gregwhitworth** [comment]
@ststimac re-ping on this

@brianespinosa thanks for the offer, we'll definitely need the help as the few that have common contribs are pretty full on spec modifications. @ststimac mind re-sharing the comp here in image form, I'm fine with us having the discussion in the public since most folks don't make their way this far down in the issues list anyways :)

**Comment by ststimac** [comment]
@brianespinosa @gregwhitworth Here are the comps but w/o the considerations discussed. I'm back from vacation so hoping to carve out some time to make updates. Here were the first two initial variations. I don't recall one being picked to iterate on.

![Desktop - 2](https://user-images.githubusercontent.com/18073131/92031977-4c1d1e80-ed1e-11ea-9786-110d5f7a210b.png)

![Desktop - 1](https://user-images.githubusercontent.com/18073131/92031970-4aebf180-ed1e-11ea-82f3-158128111e2d.png)

**Comment by brianespinosa** [comment]
Thanks @ststimac these look great!

I think the second one makes more sense but maybe I am just not fully understanding the analysis section. Would love to hear the opinions of others on this. As well.

Are there considerations for other screen sizes in terms of making this responsive as well?

**Comment by ststimac** [comment]
This is linked to #287

**Comment by brianespinosa** [comment]
> Are there considerations for other screen sizes in terms of making this responsive as well?

@ststimac not sure if you saw my question referenced above. If you had considerations for responsive design, I would prefer to use those. If not, I can probably make this design responsive. I have less time to dedicate to this than I did in September, but I can probably carve out some time towards the end of this month.

**Comment by ststimac** [comment]
@brianespinosa we're starting up a separate conversation in the Discord for the site re-design. I think these wireframe-y mockups are going to get re-done as there is a larger conversation around content that is happening and organizing things.

**Comment by levithomason** [comment]
@ststimac Thanks much for creating the comps and helping push this forward. I really like the [first comp](https://user-images.githubusercontent.com/18073131/92031977-4c1d1e80-ed1e-11ea-9786-110d5f7a210b.png) there in that it calls out and dedicates a top-level section to the "Analysis" (which could also be "Research") arm of Open UI.

The reason I really like this is because when I first formed Open UI, the original mission was research alone. However, as there has been more activity on the platform side than research side, I do worry about the research arm getting lost a bit in the mission. Again, this is why I really like the first comp there, the research aspect is called out top-level.

Both look fantastic BTW. I'm speaking mostly to the representation of what Open UI is about when I say I like first sample over the second.

**Comment by brianespinosa** [comment]
@ststimac  Ah, I did not know there was a discord. If someone else is wanting to throw their own resources at the implementation side, I can let them take it. But if the action item for making this happen is not assigned, I am still happy to put some time into this once we have more info on that direction. I did not see anything about a Discord in the docs. I am happy to join it if someone thinks that would be helpful.
